features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/source/src/app/components/Apis/Details/Endpoints/GeneralConfiguration/EndpointSecurity.jsx

```diff
@@ -85,56 +91,50 @@ function EndpointSecurity(props) {
                     ))}
                 </TextField>
             </Grid>
-            <Grid item container xs={12} spacing={4}>
-                <Grid item>
-                    <TextField
-                        disabled={isRestricted(['apim:api_create'], api)}
-                        required
-                        variant='outlined'
-                        id='auth-userName'
-                        label={(
-                            <FormattedMessage
-                                id='Apis.Details.Endpoints.GeneralConfiguration.EndpointSecurity.user.name.input'
-                                defaultMessage='User Name'
-                            />
-                        )}
-                        onChange={
-                            event => setEndpointSecurityInfo({ ...endpointSecurityInfo, username: event.target.value })}
-                        value={endpointSecurityInfo.username}
-                        onBlur={(event) => { onChangeEndpointAuth(event, 'username'); }}
-                    />
-                </Grid>
-                <Grid item>
-                    <TextField
-                        disabled={isRestricted(['apim:api_create'], api)}
-                        required
-                        variant='outlined'
-                        type='password'
-                        id='auth-password'
-                        label={(
-                            <FormattedMessage
-                                id='Apis.Details.Endpoints.GeneralConfiguration.EndpointSecurity.password.input'
-                                defaultMessage='Password'
-                            />
-                        )}
-                        value={endpointSecurityInfo.password}
-                        onChange={
-                            event => setEndpointSecurityInfo({ ...endpointSecurityInfo, password: event.target.value })}
-                        onBlur={(event) => { onChangeEndpointAuth(event, 'password'); }}
-                    />
-                </Grid>
+            <Grid item xs={6}>
+                <TextField
+                    disabled={isRestricted(['apim:api_create'], api)}
+                    required
+                    fullWidth
+                    variant='outlined'
+                    id='auth-userName'
+                    label={(
+                        <FormattedMessage
+                            id='Apis.Details.Endpoints.GeneralConfiguration.EndpointSecurity.user.name.input'
+                            defaultMessage='User Name'
```

Comment:
Shouldn't this be one word?

Reply:
Yes... 
Fixed in https://github.com/wso2/carbon-apimgt/pull/7535/commits/c22440425544ac39b82a446c08ad257caa7f90cb